frontend/src/app/office-hours/office-hours.resolver.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * The Office Hours Resolver allows office hours sections and events to be injected into the routes
+ * of components.
+ *
+ * @author Sadie Amato, Madelyn Andrews, Bailey DeSouza, Meghan Sun
+ * @copyright 2024
+ * @license MIT
+ */
+
+import { inject } from '@angular/core';
+import { ResolveFn } from '@angular/router';
+import { OfficeHoursSectionDetails } from './office-hours.models';
+import { OfficeHoursService } from './office-hours.service';
+import { catchError, of } from 'rxjs';
+import { Term } from '../academics/academics.models';
+import { AcademicsService } from '../academics/academics.service';
+
+/** This resolver injects a section into the Office Hours Section component. */
+export const sectionResolver: ResolveFn<
```

Comment:
Again, would recommend renaming this to `ohSectionResolver`, since `sectionResolver` already exists in the academics module!

Reply:
renamed as ohSectionResolver!